modules/framework-cats/jvm/src/test/scala/MetaJVM.scala

```diff
@@ -132,7 +132,7 @@ object MetaJVM {
     }
   }
 
-  abstract class LazyAccessSequential(global: GlobalRead, index: Int)
+  abstract class LazyAccessSequential(global: GlobalRead)
```

Comment:
This `index` stopped being used as of [this commit](https://github.com/typelevel/weaver-test/commit/d94e9f48fff8cc632fe29d763760401197fce2c5) to improve the sequential access tests. We can safely get rid of it.